api/registrations/views.py

```diff
@@ -424,3 +424,15 @@ def get_object(self):
         ], 'self': node}
         self.check_object_permissions(self.request, obj)
         return obj
+
+
+class RegistrationViewOnlyLinksList(NodeViewOnlyLinksList, RegistrationMixin):
+
```

Comment:
I think you'll need some registrations-based scopes here (and you'll probably have to define those as well).